samples/apps/org.wso2.carbon.uuf.sample.features-app/pom.xml

```diff
@@ -43,29 +43,29 @@
         <dependency>
             <groupId>org.wso2.carbon.uuf.sample</groupId>
             <artifactId>org.wso2.carbon.uuf.sample.foundation</artifactId>
-            <version>1.0.0-SNAPSHOT</version>
+            <version>${carbon.uuf.version}</version>
```

Comment:
these dependencies should go to its parent, then you dont need to define the versions here.